api/src/main/resources/db/migration/V6__add_filter_options_test_data.sql

```diff
@@ -0,0 +1,9 @@
+TRUNCATE filter_options cascade;
+ALTER SEQUENCE filter_options_id_seq RESTART WITH 1;
+
+insert into filter_options (id, field, field_type)
```

Comment:
Don´t instert test-data via Flyway. 
only add database schema and add data via manual SQL insert to the DB